### DISCUSSION TITLE
Change permissions on a Repository

### DISCUSSION THREAD

**Discussion by JimBonner** [community/community#70058] [Repositories] [answered]
### Select Topic Area

Question

### Body

I am new to GitHub and I am having problems with Push and Commit on repository "JimBonner/OrderPilot" which I created.  Getting access errors with both Push And Commit" such as unable to write to repository and in some cases unable to read from repository.  Cannot find out how to simply change the permissions on the above repository so I (the Owner) can have read/write access.  Back in the day this was no problem for me, oh well things change.  Can/would any one help me on this problem.

Thanks,
Jim B
[redacted due to PII]

**Comment by 2bitninja** [comment, marked as answer]
Hi @JimBonner I'm new to GitHub too.
Have you setup your "Personal Access Tokens" ? You can't use your username and password.

**Reply by JimBonner** [reply]
Thanks for the info.  I have not setup my personal access tokens.

**Reply by RafliKamal** [reply]
Thank you for the information

**Comment by itsdotnet** [comment]
If you are new to Github 
Firstly you should set SSH keys
If you provide other informations about Errors
I will give other advices (few days ago i got all errors in pushing to github :) so i have big expirence in this.)

**Comment by ginawyllie** [comment]
If you're facing access errors when trying to push or commit to your GitHub repository "JimBonner/OrderPilot" that you own, here are some steps to resolve the issue:

Check Authentication: Ensure you are using the correct GitHub credentials (username and password or a personal access token) for your repository.

Repository Settings: Go to your repository on GitHub and click on "Settings." Under the "Manage access" section, confirm that you have the necessary permissions.

Collaborators: If you're the owner and want to add collaborators with read/write access, you can invite them from the "Manage access" section in repository settings.

Access Token: If you are using a personal access token, verify that it has the required permissions to push and commit to the repository.

Branch Protection: Check if branch protection rules are in place for your repository. These rules might limit who can push to certain branches.

Git Configuration: Make sure your local Git configuration is set up correctly, including the correct remote repository URL.

HTTPS/SSH: If you're using HTTPS, consider switching to SSH for a more secure and reliable connection.

If you continue to face issues, it may be helpful to provide more specific details about the error messages you're encountering, as they can offer insights into the problem. Additionally, GitHub's support resources and [community](https://www.wjsmithconstruction.com/) forums can be valuable for troubleshooting such issues further.
